src/pocketmine/Player.php

```diff
@@ -2980,6 +2980,14 @@ public function sendTip($message){
 		$pk->message = $message;
 		$this->dataPacket($pk);
 	}
+	
+	public function whisper($from, $message){
```

Comment:
Actually, it would be better to call it `sendWhisper` in consistency with other similar methods.